setup.py

```diff
@@ -42,7 +42,7 @@
             'packaging',
             'pycodestyle',
             'pytest>=4',
-            'pytest-cov',
+            'pytest-cov<=2.8.1',
```

Comment:
What problems does the new version have?

Reply:
https://travis-ci.org/github/satellogic/telluric/jobs/702534719